clerk/tokens_options_test.go

```diff
@@ -8,27 +8,65 @@ import (
 
 func TestWithAuthorizedPartyNone(t *testing.T) {
 	opts := &verifyTokenOptions{}
-	WithAuthorizedParty()(opts)
+	err := WithAuthorizedParty()(opts)
 
-	assert.Len(t, opts.authorizedParties, 0)
+	if assert.Nil(t, err) {
+		assert.Len(t, opts.authorizedParties, 0)
+	}
 }
 
 func TestWithAuthorizedPartySingle(t *testing.T) {
 	opts := &verifyTokenOptions{}
-	WithAuthorizedParty("test-party")(opts)
+	err := WithAuthorizedParty("test-party")(opts)
 
-	assert.Len(t, opts.authorizedParties, 1)
-	assert.Equal(t, arrayToMap(t, []string{"test-party"}), opts.authorizedParties)
+	if assert.Nil(t, err) {
+		assert.Len(t, opts.authorizedParties, 1)
+		assert.Equal(t, arrayToMap(t, []string{"test-party"}), opts.authorizedParties)
+	}
 }
 
 func TestWithAuthorizedPartyMultiple(t *testing.T) {
 	authorizedParties := []string{"test-party", "another_party", "yet-another-party"}
 
 	opts := &verifyTokenOptions{}
-	WithAuthorizedParty(authorizedParties...)(opts)
+	err := WithAuthorizedParty(authorizedParties...)(opts)
+
+	if assert.Nil(t, err) {
+		assert.Len(t, opts.authorizedParties, len(authorizedParties))
+		assert.Equal(t, arrayToMap(t, authorizedParties), opts.authorizedParties)
+	}
+}
+
+func TestWithJWTVerificationKey(t *testing.T) {
+	key := "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAm7Zs5PFGrsrmvys1hHkSDOYoghz9+z9o+E6WgMqR+R/Af0/QRqQo/YwCmzB+01+5Us1NdSa32YuQYiMxV4T+g3eebSiBqPNiCyjl2wttCm5LAV5iHyVqwnBNcrXlA5mRFQz8lmyfpoksNDEVzJPwwHzPjKSIKsGgsrPnw6XsyOPJY/8UocscEcHptTmahHrbfNZLN0FrMneHw9tnn2AiUctuU9bw80KwPd+WFdZ6UZF/kPxVFsANJpz1aMpz7Lxi3Sz1ztUCdHvNJitRUO1Qewby4xi9DfIEECMq78LLmwGaTiKxutC6KwHLJEcbUblOJHpYVEXdBex9xGJ/2DHrBQIDAQAB"
```

Comment:
This test and the one below are currently failing with `x509: failed to parse public key (use ParsePKIXPublicKey instead for this key format`. Doing what the error suggests makes `TestClient_VerifyToken_Success_WithJWTVerificationKey` fail though.

Reply:
@chanioxaris Slowly getting there thanks to your suggestions. 😄